### PR TITLE
chore: pin input page

### PR DESCRIPTION
**Pull request by EugeneDevv**

### CODE REVIEW

**Comment by EugeneDevv** [comment]
em3

**Comment by EugeneDevv** [comment]
m5

File: lib/presentation/onboarding/login/pages/pin_input_page.dart [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            right: 20,
+          ),
+          child: SvgPicture.asset(
+            notificationIcon,
+            width: 20,
+          ),
+        ),
+        leadingWidget: mediumHorizontalSizedBox,
+      ),
+      body: SafeArea(
```
Comment (abiudrn) [inline lib/presentation/onboarding/login/pages/pin_input_page.dart:41]: this `SafeArea` widget is not necessary. remove it

File: lib/presentation/onboarding/login/pages/pin_input_page.dart [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        child: Container(
+          color: AppColors.timeLineBackgroundColor,
```
Comment (abiudrn) [inline lib/presentation/onboarding/login/pages/pin_input_page.dart:43]: the scaffold has a `backgroundColor` property instead. use it and remove this container

File: lib/presentation/onboarding/login/pages/pin_input_page.dart [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                  lockIcon,
+                  width: 30.0,
+                  height: 40.0,
+                ),
+                veryLargeVerticalSizedBox,
+                Column(
+                  children: <Widget>[
+                    Container(
+                      margin: EdgeInsets.only(
+                          bottom: MediaQuery.of(context).size.height * 0.009),
```
Comment (abiudrn) [inline lib/presentation/onboarding/login/pages/pin_input_page.dart:58]: this is a bit too specific. use a general value like 10.

Comment (abiudrn) [reply lib/presentation/onboarding/login/pages/pin_input_page.dart:58]: Apply this change for the other calculated fields in this page

File: lib/presentation/onboarding/login/widgets/key_pad_widget.dart [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+            widget.pinController.text = widget.pinController.text + buttonText;
+            widget.onChange(widget.pinController.text);
+          }
+        },
+        child: Container(
+          alignment: AlignmentDirectional.center,
+          child: Text(
+            buttonText,
+            textAlign: TextAlign.center,
+            style: const TextStyle(
```
Comment (abiudrn) [inline lib/presentation/onboarding/login/widgets/key_pad_widget.dart:136]: use `TextThemes` for text styling instead

**Review by abiudrn — CHANGES_REQUESTED** [review]

**Comment by abiudrn** [comment]
General comments:

1. I haven't seen an implementation for `Too many retries`
2. The appbar UI seems off according to the original design

**Comment by EugeneDevv** [comment]
too_manytries

**Comment by EugeneDevv** [comment]
wrong_pin

**Pull request merged**